specification/VRMC_springBone-1.0_draft/schema/VRMC_springBone.joint.schema.json

```
@@ -1,16 +1,24 @@
 {
-  "title": "SpringSetting",
+  "title": "SpringBoneJoint",
   "type": "object",
   "description": "A bone group of VRMCSpringBone.",
```

[MUST] description、こんな感じでいかがでしょう

```suggestion
  "description": "A joint of VRMCSpringBone.",
```